web/src/app/components/share-access-control/share-access-control.component.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2023 The Ground Authors.
+ * Copyright 2025 The Ground Authors.
```

Comment:
Please add missing TSDoc to this class while you're here.

Reply:
And rename to something more specific to its purpose `GeneralAccessControlComponent`?